pkg/assets/create/create_test.go

```diff
@@ -102,17 +102,16 @@ func TestEnsureManifestsCreated(t *testing.T) {
 	}
 
 	// Missing discovery info for kubeapiserverconfig
-	out := &bytes.Buffer{}
 	operatorResource := resourcesForEnsure[0]
 	resourcesForEnsure = resourcesForEnsure[1:]
 	ctx, cancel = context.WithTimeout(context.Background(), 1*time.Second)
 	defer cancel()
-	err = EnsureManifestsCreated(ctx, "testdata", nil, CreateOptions{Verbose: true, StdErr: out})
+	err = EnsureManifestsCreated(ctx, "testdata", nil, CreateOptions{Verbose: true})
 	if err == nil {
 		t.Fatal("expected error creating kubeapiserverconfig resource, got none")
 	}
-	if !strings.Contains(out.String(), "unable to get REST mapping") {
```

Comment:
this was in fact testing that the messages are seen with verbose:true ;-) the error is just aggregation on those messages.

Reply:
> this was in fact testing that the messages are seen with verbose:true ;-) the error is just aggregation on those messages.

As I discuss in the commit message, I think the returned error is more important.  But I can update this to test both.